src/src/js/grids/vuetify/dataAdaptor.ts

```diff
@@ -4,9 +4,11 @@ import type { Query, OrderBy } from "@js/grids/vuetify/types";
 
 export class DataAdaptor extends ObApiClient {
     private readonly errorCallback: Function;
+    private readonly _parentId?: string;
```

Comment:
The underscore for private fields is an outdated convention...
It should be used only in case of possible member conflict:

https://stackoverflow.com/questions/47747809/typescript-underscore-convention-for-members
https://www.typescriptlang.org/docs/handbook/2/classes.html#private